.gitignore

```diff
@@ -17,19 +11,26 @@ parts/
 src/*
 test.plone_addon/
 var/
+# do not ignore
+!.coveragerc
+!.editorconfig
+!.gitattributes
+!.gitignore
+!.gitkeep
+!.travis.yml
```

Comment:
Why would you *not ignore* stuff that is not ignored anyway? The only thing that needs to be here is `!src/collective` since there is a glob ignore on `src/*` above.